elasticsearch_django/apps.py

```diff
@@ -25,6 +25,8 @@ def ready(self):
             _connect_signals()
         else:
             logger.debug("SEARCH_AUTO_SYNC has been disabled.")
+        if 'never_auto_sync' not in settings.get_settings().keys():
```

Comment:
I've made that change. Did you want the check if never_auto_sync logic also contained in the new settings.auto_sync function? I've left that where it is, since it seemed better to do it once at the beginning rather than check every time something is saved.

Reply:
It should check everything - and in fact, it could easily be loaded into a dict on startup and stored in memory - for each model, True or False on whether to sync - cost of checking that would be immaterial, and changing settings results in a restart anyway.

Reply:
Sorry, I think what I wrote in the above comment was very unclear. The checking if models relating to individual instances are in never_auto_complete is currently in the auto_sync function. It was the logic that checks if never_auto_sync has been set at all in settings that I meant should be done once, similar to the dict idea - only just adding an empty list if it doesn't exist.